context/app/api/vitessce_confs/base_confs.py

```diff
@@ -269,20 +272,27 @@ def __init__(self, entity, nexus_token, is_mock=False, **kwargs):
         # All "file" Vitessce objects that do not have wrappers.
         super().__init__(entity, nexus_token, is_mock)
         self._base_name = kwargs["base_name"]
+        self._mask_name = kwargs["mask_name"]
+        self._image_name = kwargs["image_name"]
         self._imaging_path = f"{self.image_pyramid_regex}/{kwargs['imaging_path']}"
+        self._mask_path = f"{self.image_pyramid_regex}/{kwargs['mask_path']}"
```

Comment:
Is is generally true in this codebase that `*_path` variables are actually regexes? If that holds consistently, great... if not, maybe clean up some names, though it doesn't need to be this PR.

Reply:
I can try to clean this up - they have switched more than a few times (regexes turning into strings, strings into regex) and are used interchangeably (i.e strings as regex) so it's been hard to have consistent names for what every class uses them for.